### PR TITLE
express-hello-world - Ray Le

### PR DESCRIPTION
**Pull request by lol-ray**
image

### CODE REVIEW

File: express-hello-world/index.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const app = express();
+
+app.use(function (req, res) {
+  console.log('method:', req.method());
+  String(res.send());
```
Comment (scbowler) [inline express-hello-world/index.js:6]: No need to use the `String` constructor here. We just want to send back a sting:
```JAVASCRIPT
res.send('Hello, express');
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, but one update needed.

**Comment by lol-ray** [comment]
image

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍

**Pull request merged**